wpimath/src/main/native/cpp/estimator/DifferentialDrivePoseEstimator.cpp

```diff
@@ -19,6 +19,5 @@ DifferentialDrivePoseEstimator::DifferentialDrivePoseEstimator(
     units::meter_t leftDistance, units::meter_t rightDistance,
     const Pose2d& initialPose, const wpi::array<double, 3>& stateStdDevs,
     const wpi::array<double, 3>& visionMeasurementStdDevs)
-    : PoseEstimator<DifferentialDriveWheelPositions>(
-          kinematics, m_odometryImpl, stateStdDevs, visionMeasurementStdDevs),
+    : PE(kinematics, m_odometryImpl, stateStdDevs, visionMeasurementStdDevs),
```

Comment:
The odometry and pose estimator type abbreviations make the code much less readable. Characters are free.